packages/grafana-data/src/types/datasource.ts

```diff
@@ -472,7 +476,13 @@ export enum DataQueryErrorType {
 
 export interface DataQueryError {
   data?: {
+    /**
+     * Short information about the error
+     */
     message?: string;
+    /**
+     * Detailed information about the error
```

Comment:
```suggestion
     * Detailed information about the error. Only returned when app_mode is development.
```

Reply:
@marefr 👍 Added. Though I was getting it also when running in production mode, so maybe *sometimes* it's returned?

Reply:
Hmm are you sure? I had this code in mind https://github.com/grafana/grafana/blob/4632b070e4a4c736698df61c18494902a9f7d209/pkg/api/response/response.go#L194. Possible that this is not used for all responses I suppose?

Reply:
Lemme check again 🔎

Reply:
Oh sorry, my bad! You're totally right - `data.error` is not returned in production mode 🤦 As discussed [below](https://github.com/grafana/grafana/pull/35120#discussion_r663755749) I have changed the logic to cover this case. Thanks!